.babelrc

```diff
@@ -4,7 +4,6 @@
     "stage-2"
   ],
   "plugins": ["transform-runtime"],
-  "comments": false,
```

Comment:
Setting this to false appears to break the `[name]` part in the webpack config (e.g. `chunkFilename: '[name].[id].lazy.js',`) and the `[name]` fallsback to `[id]`.